### ISSUE TITLE
bug(a11y): language selection does not work completely reliably A4A#78

### ISSUE THREAD

**Issue by kyubisation** [lyne-design-system/lyne-components#1880]
Note: It is not entirely clear whether this is directly a problem with our `sbb-menu` or the usage of sbb.ch, but we should still check.

The language selection does not work completely reliably. If the switch is pressed again with a screen reader in the open state, then it can no longer be closed. The same problem also exists with the dropdown context menu in the connection details.

The finding occurs in FF, regardless of whether Jaws or NVDA. It cannot be reproduced on Edge, i.e. sometimes it occurs and sometimes it doesn't.

Quick start Guide
The following is a quick guide, please use FF for this:
1. Go to the top of the page, arrow down, until the language selection comes. I'm not sure what you're testing with, but I suspect NVDA. When you click on the language selection, NVDA will read out something like: "German submenu, etc.".
2. The focus is set to the chosen language, which is great. Now you press the up arrow once and you land on the extended language switch (language button to open the menu), try to close it with space or enter, that won't work. If you click enter or the spacebar once, the escape key no longer works either.

https://dev-www.sbb.ch/en

**Comment by DavideMininni-Fincons** [comment]
Replicated on Firefox + NVDA

Also replicated a similar behavior on the sbb-navigation with Chrome and Voiceover this way:
- go to https://dev-www.sbb.ch/it (it's the same on https://lyne-storybook.app.sbb.ch/?path=/docs/pages-home--docs)
- press tab until you arrive on the menu
- press enter to open the navigation 
- press tab to arrive the first element of the main navigation
- press enter to open the sub-navigation
- press tab to go to the second element in the sub-navigation list
- now press arrow left; you can do it many times in order to exit the sub-navigation menu, then exit the navigation and so on until you arrive on the banner 
- if you arrive on the language dropdown you could press enter and it navigate/press enter to select a new language
